blog/_posts/2022-09-29-scala-survey-2022-announcement.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog-detail
+post-type: blog
+by: Kacper Korban, VirtusLab
+title: Participate in the Scala Survey 2022!
+---
+
+![Scala Survey 2022 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2022-announcement.png)
+
+## The Scala Team wants to hear from you!
+
+We want to make the language we all love as amazing as it possibly can be. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we are asking you to help by taking part in the Scala Survey 2022.
+
+Survey: [Scala Survey 2022](https://scala-survey-2022.virtuslab.com/)
+
+## About the survey
+
+Completing the survey should take 4-12 minutes. The survey is anonymous and most of the questions are optional, so if you don’t have that much time, you don’t have to answer all of them.
+
+**Who is the survey intended for?** Everyone. Whether you’re an experienced Scala Developer or just starting your Scala journey, we count on your participation.
+
+After the survey closes, we will publish the aggregated results on the [Scala blog](https://www.scala-lang.org/blog/).
+
+## What else can I do to help?
+
+Spread the word! Help us get answers from as many diverse groups as possible. So share the survey on social media, your office, your university, meetups, and other communities.
```

Comment:
> So share the survey on social media, your office, your university, meetups, and other communities

I guess grammatically other prepositions should be added, like `in your office, at your university, ...` or we could somehow reformulate this sentence to remove the need for prepositions